src/adapter.js

```diff
@@ -216,72 +217,17 @@ class TwigAdapter extends Fractal.Adapter {
     }
 
     render(path, str, context, meta) {
-        let contextAttributes = context.attributes;
-        delete(context.attributes);
-        let attributes = new AttributesObject();
+        context.attributes = new Attributes(context.attributes);
```

Comment:
(here and relevant places elsewhere)

We should treat every context variable **containing** `"attributes"` as Attributes objects, not just `attributes`.

➡️ `attributes`, `title_attributes`, `content_attributes`